Sources/SwiftRefactor/ConvertComputedPropertyToStored.swift

```diff
@@ -0,0 +1,81 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=6)
+public import SwiftSyntax
+#else
+import SwiftSyntax
+#endif
+
+public struct ConvertComputedPropertyToStored: SyntaxRefactoringProvider {
+  public static func refactor(syntax: VariableDeclSyntax, in context: ()) -> VariableDeclSyntax? {
+    guard syntax.bindings.count == 1, let binding = syntax.bindings.first,
+      let accessorBlock = binding.accessorBlock, case let .getter(body) = accessorBlock.accessors, !body.isEmpty
+    else {
+      return nil
+    }
+
+    var initializer: InitializerClauseSyntax?
+
+    if body.count > 1 {
+      let closure = ClosureExprSyntax(
+        statements: body.with(\.trailingTrivia, .newline)
```

Comment:
I think it would be good to re-use the braces from the `accessorBlock`. That way we also don’t loose any trivia (in particular comments) that are associated to those braces

```suggestion
        leftBrace: accessorBlock.leftBrace,
        statements: body,
        rightBrace: accessorBlock.rightBrace
```

Eg. this make sure we don’t loose the comment in

```swift
var x: Int {
  return 1
  // 1 is the correct number here
}
```